src/components/TimeSlider.js

```diff
@@ -517,10 +527,17 @@ export default class CountryInfo extends Component {
     languages = await getAllFNSLanguages();
   }
   componentDidMount() {
+    console.log(this.props);
+
+    const { isMobile } = this.props;
+    let { CurrentRange } = this.state;
+    CurrentRange = isMobile ? mobileRange : desktopRange;
+    console.log(CurrentRange);
```

Comment:
Deploying console.log in code doesn't seem like a good practice, maybe I'm wrong because I don't know the purpose :/

Reply:
You are right!